src/diffusers/models/transformers/transformer_sd3.py

```diff
@@ -424,8 +424,7 @@ def custom_forward(*inputs):
             # controlnet residual
             if block_controlnet_hidden_states is not None and block.context_pre_only is False:
                 interval_control = len(self.transformer_blocks) / len(block_controlnet_hidden_states)
-                interval_control = int(np.ceil(interval_control))
-                hidden_states = hidden_states + block_controlnet_hidden_states[index_block // interval_control]
+                hidden_states = hidden_states + block_controlnet_hidden_states[int(index_block / interval_control)]
```

Comment:
Please note that this is a change that is backward compatible. 

When len(self.transformer_blocks) is divisible by len(block_controlnet_hidden_states), which is 100% cases for all shipped controlnets, the logic does not change anything.

Reply:
ok! but why do we have to change then?